assets/store_descriptions/en-US/strings.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
+<strings>
+    <intro>openHAB - "empowering the smart home" - vendor and technology agnostic open source home automation</intro>
+    <whatis>openHAB is a Java-based open-source home automation platform that integrates and combines a vast range of different smart home systems and technologies into one single solution.</whatis>
+    <rules>On top of the unified abstraction layer all connected devices are available for the overarching automation rule engines and different user interfaces.</rules>
+    <supported>Supported Products</supported>
+    <bindings>Over 200 specific add-ons provide support for brands, devices, technologies and communication protocols. Examples are Z-Wave, Philips Hue, Amazon Echo, Chromecast and Sonos. Discover all available add-ons and supported devices and/or functions at: https://docs.openhab.org/addons/bindings.html</bindings>
+
+    <home_automation_solutions>• Home Automation Solutions: Z-Wave, EnOcean, Netatmo, Homematic, Insteon, ...</home_automation_solutions>
```

Comment:
While at it, why not move those item markers into the script?

Reply:
Sry, I forgot to comment on that: RTL languages have the point on the other side.

Reply:
Good point. I'd still say it's better to move it into the script. Your script can take care of the left-right-decision. 😃

Reply:
I think it's best to have them in the script. Otherwise I'd have to maintain a list in the script.